<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FilterSettings</class>
 <widget class="QDialog" name="FilterSettings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>727</width>
    <height>490</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="windowIcon">
   <iconset resource="ResourceCheckSumJack.qrc">
    <normaloff>:/new/prefix1/icon/checksumjack.png</normaloff>:/new/prefix1/icon/checksumjack.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_5">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>Filter Name</string>
         </property>
         <widget class="QWidget" name="layoutWidget">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>32</y>
            <width>261</width>
            <height>151</height>
           </rect>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <widget class="QComboBox" name="comboBox">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>30</height>
              </size>
             </property>
             <property name="font">
              <font>
               <pointsize>10</pointsize>
              </font>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="verticalSpacer">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QCommandLinkButton" name="commandLinkButton">
             <property name="text">
              <string>New Filter</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCommandLinkButton" name="commandLinkButton_3">
             <property name="text">
              <string>Remove Filter</string>
             </property>
             <property name="icon">
              <iconset resource="ResourceCheckSumJack.qrc">
               <normaloff>:/new/prefix1/icon/remove.png</normaloff>:/new/prefix1/icon/remove.png</iconset>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_4">
         <property name="title">
          <string>Exclude Folder</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QGroupBox" name="groupBox_2">
         <property name="title">
          <string>Include Filter</string>
         </property>
         <widget class="QTableWidget" name="tableWidget">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>30</y>
            <width>241</width>
            <height>171</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <pointsize>8</pointsize>
           </font>
          </property>
          <attribute name="horizontalHeaderCascadingSectionResizes">
           <bool>false</bool>
          </attribute>
          <attribute name="horizontalHeaderHighlightSections">
           <bool>true</bool>
          </attribute>
          <attribute name="horizontalHeaderStretchLastSection">
           <bool>true</bool>
          </attribute>
          <attribute name="verticalHeaderVisible">
           <bool>false</bool>
          </attribute>
          <column>
           <property name="text">
            <string>Include Filter</string>
           </property>
          </column>
         </widget>
         <widget class="QPushButton" name="pushButton">
          <property name="geometry">
           <rect>
            <x>260</x>
            <y>170</y>
            <width>31</width>
            <height>31</height>
           </rect>
          </property>
          <property name="autoFillBackground">
           <bool>false</bool>
          </property>
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="ResourceCheckSumJack.qrc">
            <normaloff>:/new/prefix1/icon/plus.png</normaloff>:/new/prefix1/icon/plus.png</iconset>
          </property>
         </widget>
         <widget class="QPushButton" name="pushButton_2">
          <property name="geometry">
           <rect>
            <x>260</x>
            <y>130</y>
            <width>31</width>
            <height>31</height>
           </rect>
          </property>
          <property name="icon">
           <iconset resource="ResourceCheckSumJack.qrc">
            <normaloff>:/new/prefix1/icon/minus.png</normaloff>:/new/prefix1/icon/minus.png</iconset>
          </property>
         </widget>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_3">
         <property name="title">
          <string>Exclude Filter</string>
         </property>
         <widget class="QTableWidget" name="tableWidget_2">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>30</y>
            <width>241</width>
            <height>171</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <pointsize>8</pointsize>
           </font>
          </property>
          <attribute name="horizontalHeaderHighlightSections">
           <bool>true</bool>
          </attribute>
          <attribute name="horizontalHeaderStretchLastSection">
           <bool>true</bool>
          </attribute>
          <attribute name="verticalHeaderVisible">
           <bool>false</bool>
          </attribute>
          <column>
           <property name="text">
            <string>Exclude Filter</string>
           </property>
          </column>
         </widget>
         <widget class="QPushButton" name="pushButton_3">
          <property name="geometry">
           <rect>
            <x>260</x>
            <y>172</y>
            <width>31</width>
            <height>31</height>
           </rect>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="ResourceCheckSumJack.qrc">
            <normaloff>:/new/prefix1/icon/plus.png</normaloff>:/new/prefix1/icon/plus.png</iconset>
          </property>
         </widget>
         <widget class="QPushButton" name="pushButton_4">
          <property name="geometry">
           <rect>
            <x>260</x>
            <y>130</y>
            <width>31</width>
            <height>31</height>
           </rect>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="ResourceCheckSumJack.qrc">
            <normaloff>:/new/prefix1/icon/minus.png</normaloff>:/new/prefix1/icon/minus.png</iconset>
          </property>
         </widget>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Save</set>
     </property>
     <property name="centerButtons">
      <bool>false</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="ResourceCheckSumJack.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>FilterSettings</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>227</x>
     <y>472</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>FilterSettings</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>295</x>
     <y>478</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>comboBox</sender>
   <signal>currentIndexChanged(QString)</signal>
   <receiver>FilterSettings</receiver>
   <slot>comboIndexChanged(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>202</x>
     <y>55</y>
    </hint>
    <hint type="destinationlabel">
     <x>89</x>
     <y>454</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>commandLinkButton</sender>
   <signal>clicked()</signal>
   <receiver>FilterSettings</receiver>
   <slot>pushButtonNewFilterClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>90</x>
     <y>136</y>
    </hint>
    <hint type="destinationlabel">
     <x>359</x>
     <y>455</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>commandLinkButton_3</sender>
   <signal>clicked()</signal>
   <receiver>FilterSettings</receiver>
   <slot>pushButtonRemoveFilterClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>56</x>
     <y>177</y>
    </hint>
    <hint type="destinationlabel">
     <x>132</x>
     <y>233</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>FilterSettings</receiver>
   <slot>pushButtonAddInclude()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>637</x>
     <y>192</y>
    </hint>
    <hint type="destinationlabel">
     <x>487</x>
     <y>232</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_2</sender>
   <signal>clicked()</signal>
   <receiver>FilterSettings</receiver>
   <slot>pushButtonDelInclude()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>650</x>
     <y>158</y>
    </hint>
    <hint type="destinationlabel">
     <x>694</x>
     <y>232</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_3</sender>
   <signal>clicked()</signal>
   <receiver>FilterSettings</receiver>
   <slot>pushButtonAddExclude()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>649</x>
     <y>416</y>
    </hint>
    <hint type="destinationlabel">
     <x>719</x>
     <y>417</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_4</sender>
   <signal>clicked()</signal>
   <receiver>FilterSettings</receiver>
   <slot>pushButtonDelExclude()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>649</x>
     <y>378</y>
    </hint>
    <hint type="destinationlabel">
     <x>721</x>
     <y>379</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>FilterSettings</receiver>
   <slot>pushButtonSaveClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>586</x>
     <y>465</y>
    </hint>
    <hint type="destinationlabel">
     <x>539</x>
     <y>484</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>tableWidget</sender>
   <signal>itemChanged(QTableWidgetItem*)</signal>
   <receiver>FilterSettings</receiver>
   <slot>inItemChanged(QTableWidgetItem*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>396</x>
     <y>168</y>
    </hint>
    <hint type="destinationlabel">
     <x>363</x>
     <y>176</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>tableWidget_2</sender>
   <signal>itemChanged(QTableWidgetItem*)</signal>
   <receiver>FilterSettings</receiver>
   <slot>exItemChanged(QTableWidgetItem*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>428</x>
     <y>357</y>
    </hint>
    <hint type="destinationlabel">
     <x>366</x>
     <y>323</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>comboIndexChanged(QString)</slot>
  <slot>pushButtonNewFilterClicked()</slot>
  <slot>pushButtonRemoveFilterClicked()</slot>
  <slot>pushButtonAddInclude()</slot>
  <slot>pushButtonDelInclude()</slot>
  <slot>cellSelected(int,int,int,int)</slot>
  <slot>pushButtonAddExclude()</slot>
  <slot>pushButtonDelExclude()</slot>
  <slot>pushButtonSaveClicked()</slot>
  <slot>inItemChanged(QTableWidgetItem*)</slot>
  <slot>exItemChanged(QTableWidgetItem*)</slot>
 </slots>
</ui>
